src/components/CommandBar/ControlOverlay/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from "react";
+import WMSOverlay from "../../WMSOverlay";
+import { FC } from "react";
+
+type controlOverlayProps = {
+    show: boolean;
+    layerName: string;
+    attribution: string;
+    setShowLayer: React.Dispatch<React.SetStateAction<number>>;
+};
+
+const ControlOverlay: FC<controlOverlayProps> = ({ show, layerName, attribution, setShowLayer }) => {
+    return (
+        <div>{show && <WMSOverlay layerName={layerName} attribution={attribution} setShowLayer={setShowLayer} />}</div>
```

Comment:
Brauch es dieses `<div>` hier? Leeren Tag `<></>` verwenden?
```suggestion
        <>{show && <WMSOverlay layerName={layerName} attribution={attribution} setShowLayer={setShowLayer} />}</>
```